src/main/java/dev/drugowick/timeseriespoc/controller/HomePageController.java

```diff
@@ -18,7 +18,7 @@ public HomePageController(UserDataService userDataService) {
         this.userDataService = userDataService;
     }
 
-    @RequestMapping
+    @RequestMapping("/")
```

Comment:
This is a change in Spring Boot 3, which now drops support for automatic matching to `something` and `something/`.